src/containers/offer-page/specialization-block/SpecializationBlock.tsx

```diff
@@ -47,6 +47,11 @@ const SpecializationBlock = <T extends CreateOrUpdateOfferData>({
   const levelOptions = Object.values(ProficiencyLevelEnum)
   const subjectError = data.category && errors.subject
 
+  const checkboxListProps =
+    (userRole as string) === 'tutor'
```

Comment:
I guess it's better to use `UserRoleEnum`
```js
const checkboxListProps =
  userRole === UserRoleEnum.Tutor
  ...
```

Reply:
Thank you a lot!Fixed